src/app/components/identities/identities.controller.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+require('../../services/identities/identities.service')
+require('./identities.constants.js')
+
+angular.module('esn-account-inbox')
```

Comment:
Please use the same format with other modules:
```suggestion
angular.module('esn.account-inbox')
```